src/main/java/com/minecolonies/coremod/blocks/ModBlocks.java

```diff
@@ -24,49 +24,48 @@
      */
 
     public static List<BlockTimberFrame> timberFrames = new ArrayList<>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this member "protected". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2386)